doc/client_lua_api.md

```diff
@@ -672,6 +672,13 @@ Call these functions only at load time!
     * Called when the local player punches a node
     * Newest functions are called first
     * If any function returns true, the punch is ignored
+* `minetest.register_on_joinplayer(func(name))`
+    * Called when a player joins the game.
+    * Gets called for all players currently on a server when joining a server.
+* `minetest.register_on_leaveplayer(func(name))`
+    * Called when a player leaves the game.
+    * Not called if the client is exiting.
```

Comment:
May needs some tweaking: `Not called if the own client is exiting.` ?
Otherwise :+1: